src/server/storage.ts

```diff
@@ -72,6 +73,71 @@ async function storeFile(filePath: string, info: MinimalStudyInfo, file: File) {
     }
 }
 
+async function retrieveFile(
```

Comment:
why is this needed vs the existing `fetchFile` function, or the `urlOrContentForFile` ?